example.py

```diff
@@ -1,15 +1,25 @@
 from streamlit_keycloak import login
 import streamlit as st
-
 st.title("Keycloak example")
-keycloak = login(
-    url="http://localhost:8080",
-    realm="myrealm",
-    client_id="myclient",
-    auto_refresh=True,
-    init_options={"checkLoginIframe": False},
-    custom_labels={"errorPopupClosed": "You closed the popup!"}
-)
-
-if keycloak.authenticated:
-    st.write(keycloak)
+
+realm = 'myrealm'
+
+user = getattr(st.session_state, 'user', None)
```

Comment:
This will break the automatic token refresh as there is no longer a link to the component if you are already logged in. I would like to keep the example more general.